### PR TITLE
Add missing subnet exports to vpc-20-private.yaml template

### PR DESCRIPTION
**Pull request by tthyer**
Several subnets were left off the exports from the templates/VPC/vpc-20-private.yaml template. Add these.

### CODE REVIEW

File: templates/VPC/vpc-20-private.yaml [hunk 1]
```diff
@@ -285,6 +285,12 @@ Outputs:
     Export:
       Name:
         !Join ['-', [!Ref 'AWS::Region', !Sub '${AWS::StackName}', 'VpcCidr']]
+  PublicSubnet:
+    Description: "SubnetId of the public subnet"
+    Value: !Ref PrivateSubnet
```
Comment (xschildw) [inline templates/VPC/vpc-20-private.yaml:290]: Should this not be '!Ref PublicSubnet' ?

**Review by xschildw — CHANGES_REQUESTED** [review]
The output for PublicSubnet references the PrivateSubnet resource.

File: templates/VPC/vpc-20-private.yaml (hunk 1 above)
Comment (tthyer) [reply templates/VPC/vpc-20-private.yaml]: Oops, yes -- fixed in 8e3ef04

**Review by xschildw — APPROVED** [review]

**Comment by tthyer** [comment]
Thanks @xschildw!

**Pull request merged**